docs/source/installation.rst

```diff
@@ -83,127 +83,81 @@ Note: not all soft dependencies of ``sktime`` are also available on ``conda-forg
 The other soft dependencies can be installed via ``pip``, after ``conda install pip``.
 
 
-Development versions
---------------------
-To install the latest development version of ``sktime``, or earlier versions, the sequence of steps is as follows:
+Development version install
+---------------------------
 
-| Step 1 - ``git`` clone the ``sktime`` repository, the latest version or an earlier version.
-| Step 2 - ensure build requirements are satisfied
-| Step 3 - ``pip`` install the package from a ``git`` clone, with the ``editable`` parameter.
+Development versions of ``sktime`` are installs from static snapshots of the ``sktime`` repository.
 
-Detail instructions for all steps are given below.
-For brevity, we discuss steps 1 and 3 first; step 2 is discussed at the end, as it will depend on the operating system.
+Installing development versions is useful for testing, but not recommended
+for development and contribution. For development and contribution, see the next section
+on full developer setup.
 
-Step 1 - clone the git repository
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-The ``sktime`` repository should be cloned to a local directory, using a graphical user interface, or the command line.
-
-Using the ``git`` command line, the sequence of commands to install the latest version is as follows:
+To install the latest development version of ``sktime``,
+you can use the ``pip`` package manager to install directly from the GitHub repository:
 
 .. code-block:: bash
 
-    git clone https://github.com/sktime/sktime.git
-    cd sktime
-    git checkout main
-    git pull
+    pip install git+https://github.com/sktime/sktime.git
```

Comment:
Will it make sense to add instruction on how to install soft dependencies with this, e.g. `forecasting` extra?

Reply:
yes, absolutely - I remember we already created content for that, do you know where that is? Why not on this page?

Reply:
It's present in point 7 here: https://pip.pypa.io/en/stable/cli/pip_install/#examples